Elements/test/ObstacleTests.cs

```diff
@@ -113,7 +113,7 @@ public void IntersectsRotatedPolygon()
 
             var result = obstacle.Intersects(polyline);
 
-            Assert.False(result);
```

Comment:
@wynged @DmytroMuravskyi This image shows this test geometrically. The `polyline` is along the edge of the rotated obstacle. This is rightly classified as an intersection by our code now, but was previously asserted to be false. Why?

![image](https://user-images.githubusercontent.com/1139788/202292184-68698554-f1a4-4e77-b0b9-819a3e4b3b73.png)